policy-service/src/main/java/uk/gov/gchq/palisade/service/policy/web/PolicyRestController.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2019 Crown Copyright
```

Comment:
```suggestion
 * Copyright 2020 Crown Copyright
```
This should probably be updated.